app/src/main/java/com/example/andresarango/aughunt/challenge/challenge_dialog_fragment/ChallengeDialogFragment.java

```diff
@@ -0,0 +1,54 @@
+package com.example.andresarango.aughunt.challenge.challenge_dialog_fragment;
+
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+
+import com.example.andresarango.aughunt.challenge.Challenge;
+import com.example.andresarango.aughunt.challenge.ChallengeActivity;
+
+public class ChallengeDialogFragment<T> extends DialogFragment {
+
+    private static final String CHALLENGE_HINT = "challenge hint";
+
+    public ChallengeDialogFragment() {
+    }
```

Comment:
delete.  default constructors are auto-generated when you do not explicitly define one.